src/test/java/com/serotonin/bacnet4j/adhoc/IpMasterTest.java

```diff
@@ -34,12 +38,15 @@ public static void main(final String[] args) throws Exception {
     }
 
     public LocalDevice createIpLocalDevice() throws Exception {
-        Network network =network = new IpNetworkBuilder()
-                .withLocalBindAddress("0.0.0.0")
-                .withBroadcast("255.255.255.255", 24)
+        Map<Integer, List<String>> interfaceDetails = IpNetworkUtils.getLocalInterfaceAddresses();
+        String ifaceName = interfaceDetails.get(0).get(0);
```

Comment:
This is very difficult to follow without referring back to the javadoc above, this is why we should use a record/class with proper field names.